src/taipy/core/config/config.schema.json

```diff
@@ -0,0 +1,400 @@
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "uniqueItems": true
+          },
+          "outputs": {
+            "description": "The output data nodes referring the result(s) data of the function to be executed.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "uniqueItems": true
+          },
+          "function": {
+            "description": "",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "PIPELINE": {
+      "description": "Pipelines",
+      "type": "object",
+      "properties": {},
+      "required": [],
+      "additionalProperties": {
+        "type": "object",
+        "properties": {
+          "tasks": {
+            "description": "The list of tasks configurations.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "uniqueItems": true
+          }
+        }
+      }
+    },
+    "SCENARIO": {
+      "description": "Scenarios",
+      "type": "object",
+      "properties": {},
+      "required": [],
+      "additionalProperties": {
+        "type": "object",
+        "properties": {
+          "pipelines": {
+            "description": "The list of pipeline configs.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "uniqueItems": true
+          },
+          "frequency": {
+            "description": "The recurrence of the scenarios instantiated from this configuration. Based on this frequency the scenarios will be attached to the right cycles.",
+            "type": "string",
+            "enum": [
+              "DAILY",
+              "WEEKLY",
+              "MONTHLY",
+              "QUARTERLY",
+              "YEARLY"
+            ]
+          },
+          "comparators": {
+            "description": "The list of functions used to compare scenarios. A comparator function is attached to a scenario's data node configuration. During the scenario comparison, each comparator is applied to all the data nodes instantiated from the data node configuration attached to the comparator.",
+            "type": "dict",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "TAIPY": {
+      "description": "Taipy",
+      "type": "object",
+      "properties": {
+        "root_folder": {
+          "type": "string"
+        },
+        "storage_folder": {
+          "type": "string"
+        },
+        "clean_entities_enabled": {
+          "enum": [
+            "True:bool",
+            "False:bool"
+          ]
+        },
+        "read_entity_retry": {
+          "description": "20:int",
+          "type": "string"
+        },
+        "repository_type": {
+          "type": "string"
+        },
+        "repository_properties": {
+          "type": "dict"
```

Comment:
object ?

Reply:
@FredLL-Avaiga 
oh we don't have dictionary in json schema? how about hash map or something that's similar in data structure term?

Reply:
dict is object

Reply:
alright